config/deploy.rb

```diff
@@ -41,9 +41,6 @@
 # update shared_configs before restarting app
 before 'deploy:restart', 'shared_configs:update'
 
-# Resque pool
-after 'deploy:restart', 'resque:pool:hot_swap'
-
```

Comment:
because this is now in the dlss-capistrano-gem and happens automagically????

Reply:
@ndushay Yeppers!